javascript/javascript1/week3/smartPhone-app.js

```diff
@@ -0,0 +1,91 @@
+// first part
+let activities = []
+function addActivity (date, activity, duration){
+
+    if (typeof date === 'string' && typeof activity === 'string' && typeof duration === 'number'){
```

Comment:
This is usually a good way to write a chain of conditions.
```suggestion
    if (
      typeof date === 'string' 
      && typeof activity === 'string' 
      && typeof duration === 'number'
    ){
```